scripts/ghutils.py

```diff
@@ -37,12 +37,71 @@ def create_issue(tag, link, org, repo, pat, label):
     )
 
 
+@click.command()
+@click.option(
```

Comment:
Options that are common between commands can be applied to the group instead of individual commands. However note that they must be specified before the command. eg: `script.py --shared-opt1 --shared-opt2=foo create_issue --create-issue-opt=bah`. This means that scripts that already use ghutils.py will need to be updated, but probably worth it to avoid repeating all the params.